oslabs-beta/DockerLocal
# Projects page

## Description:
- These changes add a new component, ProjectRepoListItem to show repos associated with their parent project. I changed the type of the activeProject  useState hook from a project object to a number since it was dependent on the projectList and causing errors. 
 
## Changes I Made:
- Import "level" style in App.sass
- Change name of RepoListItem to RepoSearchListItem to be more descriptive and fix references to RepoListItem
- Create ProjectRepoListItem component with checkbox to set state of projectList hook
- Change activeProject from object to number and fix references
- Home.tsx renders projects from the result of a dummy fetch request
- Add optional isIncluded property to Repo type 

## How to Test:
- Run application with npm start
- Click Toggle button to navigate to home page
- Sidebar should populate the repo list with repos in each project's projectRepos array
- Checking/unchecking each repo should be reflected in the state and confirmed with react dev tools

![Screen Shot 2020-06-30 at 1 17 44 PM](https://user-images.githubusercontent.com/64039601/86173006-19fc0000-bad4-11ea-8740-9c4ba1c8d9cf.png)
